src/types/WizardImplementationProps.ts

```diff
@@ -0,0 +1,41 @@
+import { ComponentProps } from 'react';
+import { JsonForms } from '@jsonforms/react';
+import { CustomFile, FileUploadProps } from '@amsterdam/bmi-component-library';
+
+export type Asset = {
+	code: string;
```

Comment:
moeten we dit splitsen? gebruiken alleen de assetnaam in de titel begrijp ik, dan zouden we af kunnen met Asset als string te definieren in WizardImplementationProps.

Reply:
of verder nog abstracten en het geen asset noemen, maar subject/title oid

Reply:
Was bestaande code, kan evt later met Fortunato worden besproken.